packages/devtools_app/test/flutter/split_test.dart

```diff
@@ -54,6 +54,20 @@ void main() {
             const Size(10, 600));
       });
 
+      testWidgets('with 0% space to first child builder',
+          (WidgetTester tester) async {
+        final split =
+            buildSplitWithBuilder(Axis.horizontal, initialFirstFraction: 0.0);
+        await tester.pumpWidget(wrap(split));
+        expect(find.byKey(_w1), findsOneWidget);
+        expect(find.byKey(_w2), findsOneWidget);
+        expect(find.byKey(split.dividerKey), findsOneWidget);
+        expect(tester.element(find.byKey(_w1)).size, const Size(0, 600));
+        expect(tester.element(find.byKey(_w2)).size, const Size(790, 600));
+        expect(tester.element(find.byKey(split.dividerKey)).size,
+            const Size(10, 600));
```

Comment:
use `equals()` matcher here and elsewhere

Reply:
Is there any value that `equals` to the meaning of the code?

When given a value for the second parameter, `expect` will use an `equals` matcher by default.

Using it explicitly would make most of these expect statements run to two lines without changing the meaning of the code.  I think that the file will be more readable as-is.

Reply:
Is there any value that `equals` to the meaning of the code?

When given a value for the second parameter, `expect` will use an `equals` matcher by default.

Using it explicitly would make most of these expect statements run to two lines without changing the meaning of the code.  I think that the file will be more readable as-is.